py2many/cli.py

```diff
@@ -367,7 +370,7 @@ def _process_one(
     Returns False if reformatter failed.
     """
     suffix = f".{args.suffix}" if args.suffix is not None else settings.ext
-    output_path = _get_output_path(filename, suffix, outdir)
+    output_path = _get_output_path(pathlib.Path(filename.name), suffix, outdir)
```

Comment:
is this change necessary? filename is already a Path.

Reply:
filename can be an absolute path, which breaks the algorithm in `_get_output_path`

Reply:
`filename.relative_to(filename.parent)` makes the logic clearer